packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFWebViewHostApi.m

```diff
@@ -0,0 +1,176 @@
+  [[self getWebViewInstance:instanceId] setCustomUserAgent:userAgent];
+}
+
+- (nullable NSNumber *)
+    webViewWithInstanceIdCanGoBack:(nonnull NSNumber *)instanceId
+                             error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  return @([self getWebViewInstance:instanceId].canGoBack);
+}
+
+- (nullable NSString *)
+    urlForWebViewWithInstanceId:(nonnull NSNumber *)instanceId
+                          error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  return [self getWebViewInstance:instanceId].URL.absoluteString;
+}
+
+- (nullable NSNumber *)canGoForwardInstanceId:(nonnull NSNumber *)instanceId
+                                        error:(FlutterError *_Nullable __autoreleasing *_Nonnull)
+                                                  error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (void)createInstanceId:(nonnull NSNumber *)instanceId
+    configurationInstanceId:(nonnull NSNumber *)configurationInstanceId
+                      error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (void)evaluateJavaScriptInstanceId:(nonnull NSNumber *)instanceId
+                    javascriptString:(nonnull NSString *)javascriptString
+                          completion:(nonnull void (^)(NSString *_Nullable,
+                                                       FlutterError *_Nullable))completion {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (nullable NSNumber *)
+    getEstimatedProgressInstanceId:(nonnull NSNumber *)instanceId
+                             error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (nullable NSString *)getTitleInstanceId:(nonnull NSNumber *)instanceId
+                                    error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (void)goBackInstanceId:(nonnull NSNumber *)instanceId
+                   error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (void)goForwardInstanceId:(nonnull NSNumber *)instanceId
+                      error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (void)loadFileUrlInstanceId:(nonnull NSNumber *)instanceId
```

Comment:
Same. And s/Url/URL/ throughout.

The repeated URL is awkward though. We can't do too much to improve it because of the param ordering, but `loadFileInWebViewWithIdentifier:fileURL:readAccessURL:error:` would be somewhat better.